Kickstarter-iOS/ViewModels/UpdateViewModel.swift

```diff
@@ -5,15 +5,14 @@ import Prelude
 import ReactiveSwift
 import Result
 
-public struct UpdateData {
-  public let project: Project
-  public let update: Update
-  public let context: Koala.UpdateContext
+fileprivate struct UpdateData {
```

Comment:
yah for sure but just wanna do the minimal work right now. or you can clean up your own mess ;)